packages/contracts/package.json

```diff
@@ -0,0 +1,142 @@
+{
+  "name": "@graphprotocol/contracts",
+  "version": "5.3.2",
```

Comment:
After you fix conflicts in #872, worth double checking we don't actually set an older version here